tests/trainer/test_trainer.py

```diff
@@ -880,6 +880,46 @@ def training_step_and_backward(split_batch, batch_idx, opt_idx, optimizer, hidde
     trainer.fit(model)
 
 
+def test_gradient_clipping_by_value(tmpdir):
+    """
+    Test gradient clipping by value
+    """
+    tutils.reset_seed()
+
+    model = EvalModelTemplate()
```

Comment:
Can you use `BoringModel` instead? We are moving away from `EvalModelTemplate` because it's slower

Reply:
WIP